src/locales/cy/translation.json

```
@@ -653,6 +653,18 @@
         "govukLinkText": "hafan GOV.UK"
       }
     },
+    "cannotChangeHowGetSecurityCodeMfaReset": {
+      "title": "Ni allwch newid sut rydych yn cael codau diogelwch",
+      "header": "Ni allwch newid sut rydych yn cael codau diogelwch",
+      "paragraph1": "Gallwch ond mewngofnodi i'ch GOV.UK One Login drwy ddefnyddio'r dull rydych eisoes wedi'i sefydlu i gael codau diogelwch.",
```

```suggestion
      "paragraph1": "Gallwch ond mewngofnodi i’ch GOV.UK One Login drwy ddefnyddio’r dull rydych eisoes wedi’i sefydlu i gael codau diogelwch.",
```
Small suggestion to avoid issue with this [test for typographically incorrect quotes](https://github.com/govuk-one-login/authentication-frontend/blob/bf410466b0fc09014b347d026c4cc15f1ff34487/src/locales/tests/locale.test.ts#L44)